RELEASE-NOTES.txt

```diff
@@ -1,4 +1,4 @@
-Release Notes -- Apache Jackrabbit Oak -- Version 1.64.0
+Release Notes -- Apache Jackrabbit Oak -- Version 1.62.0
```

Comment:
this file shouldn't be part of the PR

Reply:
Fixed it, thank you for the observation.